dashboard/test/controllers/pd/workshop_daily_survey_controller_test.rb

```diff
@@ -99,8 +99,10 @@ class WorkshopDailySurveyControllerTest < ActionDispatch::IntegrationTest
 
     test 'pre-workshop foorm survey redirects to thanks when a response exists' do
       setup_summer_workshop
-      create :pd_workshop_survey_foorm_submission, pd_workshop: @summer_workshop, user: @enrolled_summer_teacher,
-             day: 0
+      create :day_0_workshop_foorm_submission,
```

Comment:
Change here is because I refactored the foorm factories and replaced the one used before with a more robust one